pyproject.toml

```diff
@@ -15,7 +15,9 @@ classifiers = [
 dependencies = [
   "databricks-sdk>=0.18,<0.20",
   "sqlglot==21.0.0",
-  "databricks-labs-blueprint~=0.1.0"
+  "databricks-labs-blueprint~=0.1.0",
+  "networkx~=3.2.1",
+  "matplotlib~=3.4.3",
```

Comment:
don't add external libraries - every library has to be justified and approved by legal. implement datastructures directly - it's not hard.

Reply:
Sure, the acutal datastructure is implemented directly, networkx is used for Visualisation, I used them instead of graphviz, as it is not been kept active.

Reply:
Modified using graphviz now